.nuspec/Xamarin.Forms.targets

```diff
@@ -1,8 +1,13 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-	<UsingTask TaskName="Xamarin.Forms.Build.Tasks.XamlGTask" AssemblyFile="$(MSBuildThisFileDirectory)Xamarin.Forms.Build.Tasks.dll" />
-	<UsingTask TaskName="Xamarin.Forms.Build.Tasks.CssGTask" AssemblyFile="$(MSBuildThisFileDirectory)Xamarin.Forms.Build.Tasks.dll" />
-	<UsingTask TaskName="Xamarin.Forms.Build.Tasks.XamlCTask" AssemblyFile="$(MSBuildThisFileDirectory)Xamarin.Forms.Build.Tasks.dll" />
+	<PropertyGroup>
+		<_XFBuildTasksLocation Condition="'$(_XFBuildTasksLocation)' == '' AND '$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)netstandard2.0\</_XFBuildTasksLocation>
+		<_XFBuildTasksLocation Condition="'$(_XFBuildTasksLocation)' == ''">$(MSBuildThisFileDirectory)net461\</_XFBuildTasksLocation>
```

Comment:
The path seems wrong. It should be `..\net461` instead `net461`.

Reply:
The root cause of this error is target files in netstandard1.0 directory of the nuget package. 
Because the TFM directory has a higher priority, following files must be deleted.

https://github.com/xamarin/Xamarin.Forms/blob/0d1dbefc4920e137dbd295193d470225155af4bd/.nuspec/Xamarin.Forms.nuspec#L88-L91

Reply:
but they should be replaced by a switch to load the ns1.0 tasks when needed

Reply:
@StephaneDelcroix Generanlly, under build/ directory, common files are located. If you have any target framework specific target files, you should place the files including only TFM specific contents to build/{TFM}/ directory and make it include the common files. 
e.g.) build/Xamarin.Forms.targets  <-- common .targets file
build/netstandard1.0/Xamarin.Forms.targets <-- it includes common file and something more for ns1.0

Reply:
@WonyoungChoi I know. but if we drop the ns1.0/.targets, we need, in the main .targets, to pick the right assembly, just like we do for net461 vs ns2.0

Reply:
@StephaneDelcroix I got it. However, we don't need files under build/netstandard1.0 now. They are being copied unnecessarily. Even with ns1.0 compatible TFM in apps, the main target files will not be used. How about adding dummy files that import the main targets file into each TFM directory for later?
------------
build/Xamarin.Forms.DefaultItems.props
build/Xamarin.Forms.DefaultItems.targets
build/Xamarin.Forms.props
build/Xamarin.Forms.targets
build/netstandard1.0/Xamarin.Forms.props <-- imports ../Xamarin.Forms.props
build/netstandard1.0/Xamarin.Forms.targets <-- imports ../Xamarin.Forms.targets
build/netstandard2.0/Xamarin.Forms.props <-- imports ../Xamarin.Forms.props
build/netstandard2.0/Xamarin.Forms.targets <-- imports ../Xamarin.Forms.targets
build/net461/Xamarin.Forms.props <-- imports ../Xamarin.Forms.props
build/net461/Xamarin.Forms.targets <-- imports ../Xamarin.Forms.targets

Reply:
I *think* ns1.0 are used by old PCL projects. have to check with @rmarinho. I know that duplicating .targets will solve this, but I'd like to avoid it if we can.

Reply:
Yap I think we are fine.